.travis.yml

```diff
@@ -12,6 +12,22 @@ matrix:
         - TVOS_SDK=appletvsimulator12.2
         - WATCHOS_SIMULATOR='name=Apple Watch Series 4 - 44mm'
         - WATCHOS_SDK=watchsimulator5.2
+    - os: osx
+      osx_image: xcode10.2
+      env:
+        - CACHE_NAME=SWIFT5_0
+        - SWIFT_VERSION=5.0
+        - IOS_SIMULATOR='name=iPhone 6s,OS=12.2'
+        - IOS_SDK=iphonesimulator12.2
+        - TVOS_SIMULATOR='name=Apple TV 4K (at 1080p)'
+        - TVOS_SDK=appletvsimulator12.2
+        - WATCHOS_SIMULATOR='name=Apple Watch Series 4 - 44mm'
+        - WATCHOS_SDK=watchsimulator5.2    
+      before_install:
+        - rvm install "ruby-2.5.1"
+        - npm install -g danger
```

Comment:
You can remove this I think, `danger/tap/danger-swift` has `danger-js` as dependency already :)